src/ui/assets/AssetsListContainer.js

```diff
@@ -64,8 +66,10 @@ export const mapDispatchToProps = dispatch => ({
     }
     dispatch(applyAssetsFilter(filtprops));
   },
+  onResetFilterinCategories: () => dispatch(resetFilteringCategories()),
```

Comment:
It seems that there's a minor typo.
```suggestion
  onResetFilteringCategories: () => dispatch(resetFilteringCategories()),
```